src/payment/strategies/stripev3/stripev3-payment-strategy.ts

```diff
@@ -278,9 +278,9 @@ export default class StripeV3PaymentStrategy implements PaymentStrategy {
                         stripeElement = this._stripeCardElements[0];
 
                         try {
-                            cardNumberElement.mount(`#${cardNumberElementOptions.containerId}`);
-                            cardExpiryElement.mount(`#${cardExpiryElementOptions.containerId}`);
-                            cardCvcElement.mount(`#${cardCvcElementOptions.containerId}`);
+                            cardNumberElement.mount(`#${cardNumberElementOptions.elementOrSelector}`);
```

Comment:
````#${cardNumberElementOptions.elementOrSelector}````

this still seems to be an id? we are prepending `#`

Reply:
> `#${cardNumberElementOptions.elementOrSelector}`
> 
> this still seems to be an id? we are prepending `#`

Mount supports CSS Selectors, see https://stripe.com/docs/js/element/mount